Tests/Rules/UseCorrectCasing.tests.ps1

```diff
@@ -11,6 +11,19 @@ Describe "UseCorrectCasing" {
         Invoke-Formatter '"$(get-childitem)"' | Should -Be '"$(get-childitem)"'
     }
 
+    It "Corrects alias correctly" {
+        Invoke-Formatter 'Gci' | Should -Be 'gci'
+        Invoke-Formatter '?' | Should -Be '?'
+    }
+
+    It "Corrects applications on Windows to not end in .exe" -Skip:($IsLinux -or $IsMacOS) {
+        Invoke-Formatter 'Cmd' | Should -Be 'cmd'
+    }
+
+    It "Preserves extension of applications on Windows" -Skip:($IsLinux -or $IsMacOS) {
+        Invoke-Formatter 'Cmd.exe' | Should -Be 'cmd.exe'
```

Comment:
```suggestion
        Invoke-Formatter 'Cmd.exe' | Should -Be 'cmd.exe'
        Invoke-Formatter 'MORE.com' | Should -Be 'more.com'
        Invoke-Formatter 'WinRM.cmd' | Should -Be 'winrm.cmd'
        Invoke-Formatter 'CertMgr.MSC' | Should -Be 'certmgr.msc'
```

May as well add more (assuming these are all available in testing).

Also worth adding some tests for things that aren't available as commands on Windows just to make sure